src/components/PersonLink.tsx

```diff
@@ -0,0 +1,98 @@
+import classNames from 'classnames';
+import { Person } from '../types';
+import { NavLink, redirect, useLocation } from 'react-router-dom';
+
+type Props = {
+  people: Person;
+  peopleList: Person[];
+};
+
+export const PersonLink: React.FC<Props> = ({ people, peopleList }) => {
+  const location = useLocation();
+
+  const isNameInTable = peopleList
+    .map(person => person.name)
+    .includes(people.name);
+
+  const isFatherNameInTable = people.fatherName
+    ? peopleList.map(person => person.name).includes(people.fatherName)
+    : false;
+
+  const isMotherNameInTable = people.motherName
+    ? peopleList.map(person => person.name).includes(people.motherName)
+    : false;
+
+  const isFemale = (name: string) =>
+    peopleList.map(person =>
+     {return person.name === name && person.sex === 'f' ? true : false}
+    );
+
+  const findPersonByParentName = (name: string | null) => {
+    return peopleList.find(person => person.name === name) || null;
+  };
+
+  return (
+    <tr
+      data-cy="person"
+      className={classNames({
+        'has-background-warning':
+          location.pathname === `/people/${people.slug}`,
+      })}
+    >
+      <td>
+        {isNameInTable ? (
+          <NavLink
+            // href="#/people/jan-van-brussel-1714"
+            to={`/people/${people.slug}`}
+            className={classNames({ 'has-text-danger': people.sex === 'f' })}
+          >
+            {people.name}
+          </NavLink>
+        ) : (
+          <p>{people.name}</p>
+        )}
+      </td>
+
+      <td>{people.sex}</td>
+      <td>{people.born}</td>
+      <td>{people.died}</td>
+      <td>
+        {isMotherNameInTable ? (
+          <NavLink
+            to={`/people/${findPersonByParentName(people.motherName)?.slug}`}
+            className={classNames({ 'has-text-danger': isFemale })}
```

Comment:
The `className` logic for `isFemale` is incorrect because `isFemale` is a function, not a boolean. Ensure `isFemale` returns a boolean value for correct usage here.